src/test/java/pl/edu/pw/mini/ingreedio/api/service/ProductServiceTest.java

```diff
@@ -36,6 +41,10 @@ public class ProductServiceTest extends IntegrationTest {
     @Autowired
     private ProductRepository productRepository;
 
+    @Autowired
+    // TODO: make it better??
+    private UserService userService;
```

Comment:
I think i will design some Bean with mocked user that we will be injecting during tests.

Reply:
But that is part of User refactor....